.github/workflows/continous-integration.yml

```diff
@@ -155,13 +157,52 @@ jobs:
         COVERALLS_SERVICE_NAME: github
       run: poetry run coveralls
 
+  docker:
+    name: Build Docker
+    runs-on: ubuntu-latest
```

Comment:
should we add `needs: [test]`?

Reply:
Or:
- build right away and have the images as artifacts
- push on in an extra job which requires `[quality, test, ...]`

Reply:
mhm I mean this is what we had before as well (docker hub is just building everything) so I'd keep it like that for now